documentation/configuration.rst

```diff
@@ -671,4 +671,13 @@ FLEXMEASURES_API_SUNSET_LINK
 
 Allow to override the default sunset link for your clients.
 
-Default: ``None`` (defaults are set internally for each sunset API version, e.g. ``"https://flexmeasures.readthedocs.io/en/v0.13.0/api/v2_0.html"`` for v2.0)
\ No newline at end of file
+Default: ``None`` (defaults are set internally for each sunset API version, e.g. ``"https://flexmeasures.readthedocs.io/en/v0.13.0/api/v2_0.html"`` for v2.0)
+
+Perforamance optimizations
+----------------------------
+
+FLEXMEASURES_MVIEW_UPDATE_INTERVAL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Interval in minutes to refresh the materialized views in the background.
+
+Default: ``0`` (minutes)
```

Comment:
`0` probably (should) mean that no materialized views are being crested / refreshed?